.github/workflows/publish_docker.yaml

```diff
@@ -0,0 +1,58 @@
+name: Publish Container to GitHub
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  pre_job:
+    name: Pre Job
+    runs-on: ubuntu-latest
+    outputs:
+      has_tfcli_change: ${{ steps.skip_check.outputs.tfcli }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: skip_check
+        with:
+          list-files: csv
+          filters: |
+            tfcli:
+              - 'operations/docker-compose.yml'
+              - 'operations/Dockerfile'
+              - '.github/workflows/build_docker.yml'
+
+  publish_tfcli:
+    name: Publish Terraform CLI
+    needs: pre_job
+    if: ${{ needs.pre_job.outputs.has_tfcli_change == 'true' }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: operations
+    env:
+      IMAGE_NAME: cdcgov/prime-reportstream_tfcli
+    permissions:
+      contents: read
+      packages: write
```

Comment:
Magic! Settings these values ensures the `GITHUB_TOKEN` has the correct permissions.